content/docs/overview/some_terms.md

```diff
@@ -36,8 +36,37 @@ If you right away want to deep-dive into the topic, you can have a look at the [
 
 ## What are Artifacts?
 
-- `Sources`: input for creating resources e.g. source code
-- `Resources`: contain the code to “do” something, e.g. OCI Images, binaries, etc.
-- Have a component-local identity (name and version)
-- Have a `Type` (similar to a MIME-Type)
-- Have an exchangeable `Access` (formal description how to retrieve an artifact, e.g. a download URL)
+- Can be either of kind
+  - `sources`: input for creating resources e.g. source code or
+  - `resources`: contain the code to “do” something, e.g. OCI images, binaries, etc.
+- Have a component-local `Artifact Identity` (identity attribute set consists of `name`, `extraIdentity` (optional) and `version` (optional))
```

Comment:
> identity attribute set
I think the identity attributes are not a set. Also, I am unsure whether this is a good place to already be so detailed (1: there is the actual _spec_ for more details, 2: OCM Coordinates are explained in the next chapter in this document)

That being said, I also think it is confusing to list certain attributes, and talk about them being optional or (implicitly) non-optional. Also, `version` attribute is not optional as an attribute. It is just not necessarily part of the identity, even if it is present. Again: such details are maybe not well-placed in a document that aims (I think at least) towards establishing a general idea of some terms